structcol/montecarlo.py

```diff
@@ -465,6 +658,10 @@ def scat_abs_length(radius, n_particle, n_sample, volume_fraction, wavelen):
     # on the length scale being in micrometers, this should be explicitly
     # documented (or should use a function decorator to ensure that the
     # arguments have the correct dimensions)
+    
+    # RESPONSE: I converted to um because I don't know how to make pint simplify
```

Comment:
delete TODO and RESPONSE and comment the code explaining what you are doing so that someone reading the code for the first time can understand why you are converting to microns